src/main.scss

```diff
@@ -48,8 +48,8 @@ $color-state-live-draft: #43b1b0;
 $color-box-background: $color-white;
 $color-box-border: $color-grey-3;
 $color-box-text: $color-black;
-$color-textarea-background: $color-white;
-$color-textarea-border: $color-grey-3;
+$color-textarea-background: #f2fcfc;
```

Comment:
Aren't we trying to reduce the number of shades of colours in the Wagtail Admin?

Reply:
Think we can pick this up in design review